reporter/src/main/resources/templates/asciidoc/vulnerability_report.ftl

```diff
@@ -46,9 +46,7 @@ lack relevant vulnerabilities. Further details about the issues that occurred ca
 == Packages
 [#assign advisorResults = helper.advisorResultsWithVulnerabilities()]
 [#list advisorResults as id, results]
-=== ${id.name}
-
-Package URL: _${ModelExtensions.toPurl(id)}_
+=== ${ModelExtensions.toPurl(id)}
```

Comment:
Commit message: "instead **of**"
For consistency, this should actually find the package metadata by id and use the purl from there, because it could have been set in a curation. But as it was like that before that's fine with me.

Reply:
> Commit message: "instead **of**"

"for" should actually be correct here, as it's supposed to read like "use them instead, for listing the packages". I've anyway reworded the commit message.

> But as it was like that before that's fine with me.

Ok, I'll then leave it as-is for now.